config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users, path_prefix: 'app'
   resources :users
+  resources :passwords
```

Comment:
Can we restrict this to.. 
```suggestion
  resources :passwords, only: [:edit, :update]
```
So we arent adding routes for controller actions that dont exist.